src/Tutor.Core/LearnerModel/DomainOverlay/KnowledgeComponentMasteries/KnowledgeComponentMastery.cs

```diff
@@ -279,5 +289,9 @@ private static void When(SoughtHelp @event)
              * Possibly record how many times help was sought somewhere?             
              */
         }
+
+        private static void When(InstructorMessageEvent @event)
```

Comment:
Add a nested comment explaining why this method is empty, similar to what Milana did. We will find a way to merge these empty methods.